### PR TITLE
Fix some bugs in the front end page

### PR DESCRIPTION
**Pull request by Beacontownfc**
Fix some problems in the issue #3455

### CODE REVIEW

File: linkis-web/vue.config.js [hunk 1]
```diff
@@ -202,7 +202,7 @@ module.exports = {
   devServer: {
     proxy: {
       '/api': {
-        target: 'http://127.0.0.1:9001', // linkis
+        target: 'http://114.116.102.50:9001', // linkis
         changeOrigin: true,
```
Comment (casionone) [inline linkis-web/vue.config.js:206]: The actual ip needs to be removed from the code

File: linkis-web/src/apps/linkis/router.js [hunk 2]
```diff
@@ -138,116 +138,116 @@ export default [
...
+      component: () =>
+        import('./module/datasourceType/index.vue'),
+      meta: {
+        title: 'datasourceType',
+        publicPage: true,
       }
+    },
+    {
+      name: 'EnginePluginManagement',
+      path: 'EnginePluginManagement',
```
Comment (casionone) [inline linkis-web/src/apps/linkis/router.js:243]: About router  element name and path， can there be a unified naming convention?
maybe ?
```
{
      name: 'enginePluginManagement'
      path: 'enginePluginManagement'
}
```

File: linkis-web/src/apps/linkis/module/datasourceEnv/EditForm/index.vue [hunk 3]
```diff
@@ -141,15 +171,30 @@ export default {
...
+        this.rule[5].hidden = !this.formData.keytab;
         this.getData(newV)
       },
       deep: true,
     },
+    formData: {
+      handler(newV){
+        this.rule[4].hidden = newV.datasourceTypeId == 4 ? false : true;
+        if(this.rule[4].hidden) this.rule[5].hidden = true;
+        this.rule[5].hidden = !this.formData.keytab;
```
Comment (casionone) [inline linkis-web/src/apps/linkis/module/datasourceEnv/EditForm/index.vue:194]: The data source id of hive type does not have to be 4
best judged by name

**Review by jacktao007 — APPROVED** [review]
LGTM.

**Pull request merged**